node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function cp() {
+  const filePath = new URL(process.argv[2], import.meta.url);
```

Comment:
import.meta.url isn't necessary here since readFile/writeFile by default will look in the current directory.

Reply:
Updated